cfme/common/vm_views.py

```diff
@@ -34,6 +48,8 @@ def data(self):
 
             state = os.path.split(state)[1]
             state = os.path.splitext(state)[0]
+        except NoSuchElementException:
```

Comment:
this old code looks weird but I don't think you need to fix this in this PR

Reply:
Let me know the changes I will raise a PR. @izapolsk